### DISCUSSION TITLE
Warning from Resolving packages by runing the command "yarn"

### DISCUSSION THREAD

**Discussion by frabervo** [eclipse-theia/theia#12131] [Q&A] [answered]
Hello, 
When I install the dependencies using the "yarn" command, I get for example the following warnings: 
`warning lerna > @lerna/bootstrap > @npmcli/arborist > @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs`

`browser-app > @theia/filesystem > trash > uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.`

`warning "workspace-aggregator-b8f67339-ebd1-42ba-9081-ef68e29435ba > browser-app > @theia/debug > requestretry@7.1.0" has unmet peer dependency "request@2.*.*".`

How can I please ignore this warning, or should I just ignore it?

**Comment by msujew** [comment, marked as answer]
Hey @frabervo,

you can ignore these issues, yarn handles them by themselves fairly well. I've created https://github.com/eclipse-theia/theia/pull/12133 for the outdated `uuid` dependency.
